index.html

```diff
@@ -229,6 +207,27 @@ <h2>
               </ul>
           </div>
         </div>
+
+        <section class="contact-me">
+          <h3 class="contact-me-header">
+           I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.
+          </h3>
+          <form action="#" method="post" class="form-contact-me">
+            <input type="text"  name="username" placeholder="Full name" id="name" required>
+            <input type="email" name="email" placeholder="Full name" id="email" required>
+            <textarea name="comment" id="text-area" cols="30" rows="10">Enter text here</textarea>
```

Comment:
- [ ] Please make sure the message field only accepts up to 500 characters.
    - You can use this attribute `maxlength="500"`